ecr_api.py

```diff
@@ -163,7 +175,7 @@ def __init__ ( self , retries=60) :
             try:
                 self.db=MySQLdb.connect(host=mysql_host,user=mysql_user,
                   passwd=mysql_password,db=mysql_db)
-            except Exception as e:
+            except Exception as e: # pragma: no cover
```

Comment:
This is fine for now. But in the future I would like to see our error handling of database connection failures tested using `mock` to simulate calls to `MySQLdb.connect` (in success and failure conditions)